noxfile.py

```diff
@@ -100,7 +100,7 @@ def default(session):
 
     if session.python == "3.8":
         extras = "[tests,alembic]"
-    elif session.python == "3.9":
+    elif session.python in {"3.9", "3.10"}:
```

Comment:
We don't necessarily to add the `geography` extra to more than one session.

Reply:
Changed to `session.python == "3.10"`